### PR TITLE
Update workarea.cpp

### PR DESCRIPTION
**Pull request by akshitagupta20**
I rectified the error.

### CODE REVIEW

**Comment by ice0** [comment]
Please provide a link to the issue you are trying to fix.

File: synfig-studio/src/gui/workarea.cpp [hunk 1]
```diff
@@ -1469,7 +1469,7 @@ WorkArea::on_drawing_area_event(GdkEvent *event)
 
 		if (get_drag_mode() == DRAG_WINDOW) {
 			set_focus_point(get_focus_point() + mouse_pos-drag_point);
-    	} else
+    	}
```
Comment (Keyikedalube) [inline synfig-studio/src/gui/workarea.cpp:1472]: You're supposed to fix indentation not remove `else`

**Review by Keyikedalube — CHANGES_REQUESTED** [review]

**Comment by Keyikedalube** [comment]
> I rectified the error.

@akshitagupta20 Edit your description and add this line

👉 Fixes #1760 👈

**Comment by ice0** [comment]
I am closing this PR because there is no response from @akshitagupta20.

**Pull request closed without merging**